src/demo/containers/playground/FormServicesIntegrationDemo.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react'
```

Comment:
Added new demo area (note that you must change the endpoints to staging this week to see this in action).